res/controllers/Hercules DJ Console 4-Mx.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
```

Comment:
I think warning is a better approach. It is possible that some of the mapping would still work with an old version of Mixxx, which is better than nothing. The warning would let the user know that they need to upgrade to use all of the mapping.